SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,72 @@
+    |                  | Greater than                            | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Greater or Equal than                   | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less than                               | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Less or Equal than                      | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | XOR                                     | - FSS        |                    2+ |
+    |                  |                                         | - SPDZ       |                       |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Exponential                             | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Log                                     | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Max                                     | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Reciprocal                              | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Relu                                    | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Sigmoid                                 | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Softmax                                 | - FSS        |                    2+ |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Tanh                                    | - FSS        |                    2+ |
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | Public           | Multiplication                          | - Falcon     | - 3                   |
```

Comment:
Q: What does public in visibility mean here for multiplication and division

Reply:
I wanted to express that we only support public multiplication and division at the moment. Not their private counterparts.

Reply:
We have private multiplication for Falcon

Reply:
For private division in Falcon , it would be implemented in a future PR.

Reply:
We have private division for falcon, but it would be implemented in a future PR.

Reply:
True, thanks. About the private division for falcon, I think is best not to add it for now to the table. Since we are only adding what we support. Or maybe, we could add like a new table or row for "Coming" operators? I do not know.

Reply:
Sure we could leave now,add when we support it 👍